### PR TITLE
Crud for moves Part 1

### PR DESCRIPTION
**Pull request by fleawig**
I'm requesting this to be merged now so we can all continue with a functioning Move model that can save moves to database, and works with the PieceMover methods to update the board.
I started to work on Move CRUD, but creation will be dependent on Game params, so it makes sense to finish some Game CRUD first (so that moves can belong to games).
After this merge, I'll work on Game CRUD branch, then Move CRUD #2 branch.

### CODE REVIEW

**Review by nrabbi — APPROVED** [review]
Fixed the build errors that would have made it fail the codeship test. Overall, it looks good. Approved!

**Comment by fleawig** [comment]
Updated to use integer input and the Position constructor. Everything works. Good teamwork, folks! Please review so we can merge.

File: app/controllers/games_controller.rb [hunk 1]
```diff
@@ -13,22 +13,15 @@ def index; end
...
+    moves << move0 
+
+    move1 = Move.new(from: "1", to: "17")
     moves << move1
 
-    move2 = Move.new(pieces[2].position.x, pieces[2].position.y)
-    move2.to.x = move2.from.x
-    move2.to.y = move2.from.y + 2
+    move2 = Move.new(from: "2", to: "18")
     moves << move2
```
Comment (rongg) [inline app/controllers/games_controller.rb]: Why are these from and to's in quotes? They should be integers

**Review by rongg — CHANGES_REQUESTED** [review]

**Comment by rongg** [comment]
I fixed the Games table so the status works with the enum in the Game model. You can create and save a game like this:

`game = Game.new(player_1_id: 1, player_2_id: 2, status: :created)`
 `game.save`

**Comment by fleawig** [comment]
Since the requested change was so straightforward, and has been made, I'm going to merge the branch.

**Comment by fleawig** [comment]
Also, thanks for changing the Game model, Ron!

**Pull request merged**